server/api/api.go

```diff
@@ -1635,7 +1635,7 @@ func (a *API) handleDeleteSubscription(w http.ResponseWriter, r *http.Request) {
 	//   description: Workspace ID
 	//   required: true
 	//   type: string
-	// - name: BlockID
+	// - name: blockID
```

Comment:
Not related issue, that I had to fix to generate the swagger documentation.

Reply:
I forgot to re-generate the the swagger files when I checked that in...  otherwise I would have caught that.

Reply:
Yes, maybe the CI should check that, just "generate and see if something changes or not"